arm/Microsoft.AAD/DomainServices/readme.md

```diff
@@ -105,13 +108,34 @@ Tag names and tag values can be provided as needed. A tag can be left without a
 
 ## Considerations
 
-The network security group has to be created prior to running this module and assign to the Azure ADDS Subnet and associating a route table to the same subnet is not recommended.
-
-## Get base64encoded code from pfx
-Follow the below powershell commands to generate the base64 encoded code from a pfx file:
-```powershell
-  $file=get-content "<<pfx certificate file path>>" -encoding byte
-  [System.Convert]::ToBase64String($file) | Out-File pfx-encoded-bytes.txt
+- A network security group has to be created and assigned to the designated AADDS subnet before deploying this module
```

Comment:
Is this whole considerations section something we're willing to maintain?

Reply:
I guess.
Back then I removed all consideration sections from all modules, I was asked by the team to add them again.